routes.go

```diff
@@ -305,9 +304,9 @@ func fixV2Duration(getLogger ancla.GetLoggerFunc, config ancla.TTLVConfig, v2Han
 			// if this is v2, we need to unmarshal and check the duration.  If
 			// the duration is bad, change it to 5m and add a header. Then use
 			// the v2 handler.
-			logger := getLogger(r.Context())
+			logger := sallusthttp.Get(r)
 			if logger == nil {
-				logger = log.NewNopLogger()
+				logger = sallust.Default()
```

Comment:
Application-layer code shouldn't have logic that determines default services, like logging.  The problem is that every application will do this differently, e.g. what `tr1d1um` does for default logging might not match what `scytale` does.

Application code should just panic if not supplied a logger (or any service).  It's up to the application programmer to supply things like `sallust.Default()`.